src/language-markdown/unified-plugins/front-matter.js

```diff
@@ -1,6 +1,9 @@
 import parseFrontMatter from "../../utils/front-matter/parse.js";
 
-function frontMatter() {
+/**
+ * @type {import('unified').Plugin<[], import('unified').Settings>}
+ */
+const frontMatter = function () {
```

Comment:
Does this have to be arrow function?

Reply:
Yes. Because we need using `this` in function body